src/core/utils/useReservations.tsx

```diff
@@ -0,0 +1,100 @@
+import { useGetReservationsV2 } from "../fbs/fbs";
+import { useGetV1UserReservations } from "../publizon/publizon";
+import { getQueuedReservations } from "./helpers/general";
+import {
+  mapFBSReservationToReservationType,
+  mapPublizonReservationToReservationType
+} from "./helpers/list-mapper";
+import { getReadyForPickup } from "../../apps/reservation-list/utils/helpers";
+import { ReservationType } from "./types/reservation-type";
+
+type Reservations = {
+  readyToLoan: ReservationType[];
+  queued: ReservationType[];
+  isLoading: boolean;
+  isError: boolean;
+};
+
+type UseReservationsType = {
+  all: Reservations & {
+    reservations: ReservationType[];
```

Comment:
Consider adding `reservations` to the `Reservations` type.

1. This makes the types consistent.
2. As far as I can tell there is no way to retrieve all reservations from `fbs` or `publizon`. I cannot tell why we would not want that.